### PR TITLE
Add link to logo

### PR DESCRIPTION
**Pull request by khd33j**

### CODE REVIEW

File: Client/src/Components/Sidebar/index.jsx [hunk 1]
```diff
@@ -162,62 +162,69 @@ function Sidebar() {
...
+            </Typography>
+          </Stack>
+          <IconButton
             sx={{
-              position: "relative",
-              backgroundColor: theme.palette.primary.main,
-              borderRadius: theme.shape.borderRadius,
-              userSelect: "none",
+              position: "absolute",
+              top: 60,
```
Comment (ajhollid) [inline Client/src/Components/Sidebar/index.jsx:200]: Why was this changed to `position: absolute`?

**Review by ajhollid — CHANGES_REQUESTED** [review]
When submititng a PR please include a summary of your changes in checkbox format like so:

- [x] This PR adds a link to the logo
  - [x] Changed position from relative to absolute because .....
  - [x] Added link to icon
  
  etc.
  
Reviewers need to be able to very quickly look at what your PR is and what it is doing, we have limited time available to review pull requests.  Thanks!

**Comment by khd33j** [comment]
@ajhollid I'm sorry about that. I made a mistake and indent the IconButton code too. 
Thanks for the feedback. 

I'll open another PR with the appropriate changes and include a summary for the changes too.
